webapp/templates/base.html.twig

```diff
@@ -7,6 +7,8 @@
     <link rel="icon" href="{{ asset('favicon.ico') }}">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+    <link rel="stylesheet" href="{{ asset("css/blog.css") }}">
```

Comment:
podobne jako je tady funkce asset, tak muzeme volat asset ("media/image/blog/123.jpg")

Castokrat se prave css a js davaji mimo (do podlslozek) a potom i media se daji do podslozky - z media se totiz pak stane docker volume
- sem uzivatele nahravaji svuj obsah, neni to v codebase projektu